src/netspresso_trainer/cfg/augmentation.py

```diff
@@ -1,12 +1,20 @@
 from dataclasses import dataclass, field
 from pathlib import Path
-from typing import Optional, Union
+from typing import List, Optional, Union
 
 from omegaconf import MISSING, MissingMandatoryValue
 
+DEFAULT_IMG_SIZE = 256
+
 
 @dataclass
-class ColorJitter:
+class Transform:
+    name: str = MissingMandatoryValue
```

Comment:
@illian01 

`MissingMandatoryValue` is error type...

See https://omegaconf.readthedocs.io/en/2.1_branch/usage.html#mandatory-values

Reply:
Oops, sorry. I fixed it.